app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +27,14 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
```

Comment:
Then would I just use the params thing, or would I still have to go back to @deploy?